### ISSUE TITLE
Past proceedings URLs are broken

### ISSUE THREAD

**Issue by matthewfeickert** [scipy-conference/conference.scipy.org#55]
One of the projects I work on uses `sphinx`'s `linkcheck` to validate all the URLs in our docs. We noticed that the URLs that link to the conference proceedings have started to fail the `linkcheck`

```
...
2024-03-10T06:07:22.7942075Z (        outreach: line   60) broken    http://conference.scipy.org/proceedings/scipy2019/slides.html - 404 Client Error: Not Found for url: https://conference.scipy.org/proceedings/scipy2019/slides.html
2024-03-10T06:07:22.9437406Z (       citations: line   25) ok        https://arxiv.org/abs/2103.12504
2024-03-10T06:07:22.9849483Z (        outreach: line   39) broken    http://conference.scipy.org/proceedings/scipy2020/slides.html - 404 Client Error: Not Found for url: https://conference.scipy.org/proceedings/scipy2020/slides.html
...
```

and currently all the links on https://conference.scipy.org/proceedings/ (e.g. https://conference.scipy.org/proceedings/scipy2023 fail with a 404).

**Comment by achabotl** [comment]
The proceedings pages were hosted on an Enthought-owned S3 bucket. When we changed the DNS to publish this site to Github pages, the proceedings links broke.

I made a local copy of all the proceedings pages, but we have a  problem due to [GitHub Pages limits](https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages#usage-limits). A Pages size can have a max size of 1GB and all proceedings add up to ~900 MB.

Looks like we'll have to go the S3 route.

We [can't transfer ownership](https://stackoverflow.com/questions/29822570/how-to-transfer-an-amazon-s3-bucket-to-another-account) of a bucket, only of objects. In fact, the recommended thing to do is to sync two buckets, effectively copying all the files.

Investigation continues.

**Comment by achabotl** [comment]
I'll temporarily try to write a redirect to missing pages until we have a solution.

**Comment by achabotl** [comment]
A minimal redirect like this doesn't work by default. Pelican [wants more metadata](https://docs.getpelican.com/en/stable/content.html)

```
<!DOCTYPE html>
<html>
  <head>
    <meta http-equiv="refresh" content="0; url='http://conference.scipy.org.s3-website-us-east-1.amazonaws.com/proceedings/scipy2023/'" />
  </head>
  <body>
    <p>You will be redirected soon!</p>
  </body>
</html>
```

**Comment by matthewfeickert** [comment]
Reopening given PR #56 isn't enough to fully close.

**Comment by achabotl** [comment]
@matthewfeickert I think this will be "fixed" by #56. That PR is not the _real_ solution, but it's better than completely broken.

**Comment by matthewfeickert** [comment]
Thanks very much, @achabotl!

> That PR is not the real solution, but it's better than completely broken.

Yes, agreed. We can close this one and move watching over to Issue https://github.com/scipy-conference/conference.scipy.org/issues/58 :+1:
